packages/victory-core/src/victory-util/helpers.js

```diff
@@ -222,13 +222,10 @@ function reduceChildren(
       const childName = child.props.name || `${childRole}-${names[index]}`;
       if (child.props && child.props.children) {
         const childProps = assign({}, child.props, pick(parentProps, sharedProps));
-        const nestedChildren =
-          child.type && isFunction(child.type.getChildren)
-            ? child.type.getChildren(childProps)
-            : React.Children.toArray(child.props.children).map((c) => {
-                const nestedChildProps = assign({}, c.props, pick(childProps, sharedProps));
-                return React.cloneElement(c, nestedChildProps);
-              });
+        const nestedChildren = React.Children.toArray(child.props.children).map((c) => {
```

Comment:
This is the issue for stacking. The domain won't be correctly calculated if `VictoryStack` `getChildren` isn't called when calculating the domain, because the y0 / y1 values wont be set on the children. I'm guessing removing this is also where we're getting most of our perf improvements, though. It's probably worth writing a specialized method for calculating a domain for VictoryStack.

Reply:
Thanks for looking into this!  I'll start poking around to see what I can figure out 👍

Reply:
Hmm... we also have to make sure that `VictoryChart` and `VictoryGroup` actually use `VictoryStack`'s `getDomain` method. Maybe it should be a static on that class the same way it is on components like `VictoryBar`

Reply:
I'm actually looking into potentially having `VictoryChart` and `VictoryGroup` properly calculate `y0` and `y1` in their `getCalculatedProps` so we can bank on the those values being available/memoized on subsequent/child `getCalculateProps` calls.

Reply:
Oh, I'm using the wrong terms... I mean `_y0` and `_y1`. Values that get added to _each_ data point when we stack values

Reply:
I haven't had a ton of time to spend on this since last week, but I did figure out a small win to fix the regression (I think, still need to check chromatic).  It does decrease performance, from 690 milliseconds to about 1.07 seconds, and also increased the number of `reduceChildren` from 21 to 771, but that is significantly better than the original 7234 calls.